### PR TITLE
Chip feedback and small Auction polish

### PR DESCRIPTION
**Pull request by Fueredoriku**
- Add audio for collecting chips
- Add audio for auctioneer sales
- Fix lighting in grand canyon for better chip look there
   - Changes skybox and upps light intensity for better bounced lighting, removed extre directional light from scene (no longer needed)
- Add curly curtain material to auction
image

TODO
- Someone, at some point, must bake the new lightmap in the grand crevice scene

### CODE REVIEW

File: Assets/Scripts/Auction/Auctioneer.cs [hunk 1]
```diff
@@ -12,6 +12,8 @@ public class Auctioneer : MonoBehaviour
     [SerializeField]
     private AudioSource audioSource;
     [SerializeField]
+    private AudioSource audioSourceFX;
```
Comment (toberge) [inline Assets/Scripts/Auction/Auctioneer.cs:15]: This is for the kaching sound? Please reconsider the name

File: Assets/Scripts/Interactables/CollectableChip.cs [hunk 2]
```diff
@@ -22,6 +24,8 @@ private void OnTriggerEnter(Collider other)
         if (!other.gameObject.TryGetComponent<PlayerManager>(out PlayerManager player))
             return;
         player.identity.UpdateChip(1);
-        Destroy(gameObject);
+        audioSource.Play();
+        GetComponent<Collider>().enabled = false;
+        chipModel.SetActive(false);
```
Comment (toberge) [inline Assets/Scripts/Interactables/CollectableChip.cs:29]: You could still destroy the coin after the audio has played

File: Assets/Scripts/Auction/Auctioneer.cs [hunk 3]
```diff
@@ -65,6 +67,7 @@ public void Sell()
         audioSource.Stop();
         audioSource.clip = sold.RandomElement();
         audioSource.Play();
+        audioSourceFX.PlayDelayed(0.5f);
```
Comment (toberge) [inline Assets/Scripts/Auction/Auctioneer.cs:70]: This delay seems just a little too long, perhaps shorten it by 100 ms?

File: Assets/Scripts/Gamestate/PlayerManager.cs [hunk 4]
```diff
@@ -266,7 +266,7 @@ private void SetLayerOnSubtree(GameObject node, int layer)
 
     private void PlayOnHit()
     {
-        if (Random.Range(0, 100) > 5)
+        if (Random.Range(0, 1000) > 5)
```
Comment (toberge) [inline Assets/Scripts/Gamestate/PlayerManager.cs:269]: bye bye colonel kernel

**Review by toberge — CHANGES_REQUESTED** [review]
The curtains look nice and it's good to have audio on chip pickup and purchase, but the volume is so low it drowns in the music so could we turn it up a little?

File: Assets/Scripts/Gamestate/PlayerManager.cs (hunk 4 above)
Comment (Fueredoriku) [reply Assets/Scripts/Gamestate/PlayerManager.cs:269]: 😔✊

File: Assets/Scripts/Interactables/CollectableChip.cs (hunk 2 above)
Comment (Fueredoriku) [reply Assets/Scripts/Interactables/CollectableChip.cs:29]: Ye, but I think the cost of running an IEnemurator waiting for the sound to finish playing is basicly the same as disabling the object like this, so I think this is the better solution for the brevity.

File: Assets/Scripts/Auction/Auctioneer.cs (hunk 1 above)
Comment (Fueredoriku) [reply Assets/Scripts/Auction/Auctioneer.cs:15]: Will do 👍

File: Assets/Scripts/Auction/Auctioneer.cs (hunk 3 above)
Comment (Fueredoriku) [reply Assets/Scripts/Auction/Auctioneer.cs:70]: Sure :)

File: Assets/Scripts/Interactables/CollectableChip.cs (hunk 2 above)
Comment (toberge) [reply Assets/Scripts/Interactables/CollectableChip.cs:29]: Destroy(gameoObject, audioSource.clip.length)
(usikker på om det er clip)
Resolve hvis du ikke vil prøve

File: Assets/Scripts/Auction/Auctioneer.cs (hunk 1 above)
Comment (toberge) [reply Assets/Scripts/Auction/Auctioneer.cs]: one should be voicelineSound and the other buySound or sth more actually meaningful but I'm not gonna fite you on dis

Comment (toberge) [reply Assets/Scripts/Auction/Auctioneer.cs]: oh actually do that because you funked up and didn't set the audiosource after rename >:)

**Review by toberge — APPROVED** [review]
leaving an approve so you can merge after fixing the stuff. sound sounds audible now

**Review by karofmah — APPROVED** [review]
Looks great! Really like the audio for collecting chips

**Pull request merged**